Tests/Functional/FunctionalTests.xml

```diff
@@ -0,0 +1,30 @@
+<phpunit
```

Comment:
Why is a custom FunctionalTests.xml needed? Normally the one from Nimut-testingframework works perfectly well.

Reply:
ok, I'll try it with nimbus configuration.